workflows/sarek.nf

```diff
@@ -153,9 +153,9 @@ pon                = params.pon                ? Channel.fromPath(params.pon).co
 
 // Initialize value channels based on params, defined in the params.genomes[params.genome] scope
 ascat_genome       = params.ascat_genome       ?: Channel.empty()
-dbsnp_vqsr         = params.dbsnp_vqsr         ?: Channel.empty()
-known_indels_vqsr  = params.known_indels_vqsr  ?: Channel.empty()
-known_snps_vqsr    = params.known_snps_vqsr    ?: Channel.empty()
+dbsnp_vqsr         = params.dbsnp_vqsr         ? Channel.value(params.known_snps_vqsr) : Channel.empty()
```

Comment:
@FriederikeHanssen and @maxulysse : The joint-germline genotyping now seems to work both when `known_snps_vqsr` etc. are set and when they are not.

Should we do a similar initialisation of the other channels in this section?